docs: add from which version public_metrics start (#671)
redpanda-data/documentation: modules/reference/pages/rpk/rpk-registry/rpk-registry-mode-set.adoc

= rpk registry mode set
// tag::single-source[]

> Set Schema registry's mode.

Reviewer: ```suggestion
Set the mode Schema Registry runs in.
```

> Running this command with no subject sets the global schema registry mode, alternatively you can use the `--global` flag to set the global schema registry mode at the same time as per-subject mode.

Reviewer: ```suggestion
Running this command with no subject sets the global mode for Schema Registry. Alternatively, use the `--global` flag to set the global mode for Schema Registry at the same time as per-subject modes.
```
And again @r-vasquez. Is the description of the global mode in the first line of this description correct?

Acceptable mode values: 

- `READONLY`

- `READWRITE`

== Usage

[,bash]
----
rpk registry mode set [SUBJECT...] [flags]
----

== Flags

[cols="1m,1a,2a"]
|===
|*Value* |*Type* |*Description*

|--global |- |Set the global schema registry mode in addition to subject modes.

|-h, --help |- |Help for set.

|--mode |string |Schema registry mode to set. Acceptable values: `READONLY`, `READWRITE` (case insensitive).

|--format |string |Output format: `json`,`yaml`,`text`,`wide`,`help`. Default: `text`.

|--config |string |Redpanda or `rpk` config file; default search paths are `~/.config/rpk/rpk.yaml`, `$PWD`, and `/etc/redpanda/redpanda.yaml`.

|-X, --config-opt |stringArray |Override `rpk` configuration settings. See xref:reference:rpk/rpk-x-options.adoc[`rpk -X`] or execute `rpk -X help` for inline detail or `rpk -X list` for terser detail.

|--profile |string |Profile to use. See xref:reference:rpk/rpk-profile.adoc[`rpk profile`] for more details.

|-v, --verbose |- |Enable verbose logging.
|===

== Examples

Set the global schema registry mode to `READONLY`:

[,bash]
----
rpk registry mode set --mode READONLY
----

> Set the schema registry mode to `READWRITE` in subjects `<foo>` and `<bar>`:

Reviewer: ```suggestion
Set the schema registry mode to `READWRITE` in subjects `<subject-1>` and `<subject-2>`:
```

[,bash]
----

> rpk registry mode set <foo> <bar> --mode READWRITE

Reviewer: ```suggestion
rpk registry mode set <subject-1> <subject-2> --mode READWRITE
```

> ----

Reviewer: Underneath, could we add a sentence. 

Replace placeholder values.
